docs/backend/tests.md

```diff
@@ -6,7 +6,7 @@ in `tests` (unit tests) and `features` (behat tests) directory.
 ## Module Folder structure
     
 ```        
-features/  Beaht tests
+features/  Behat tests
 migrations/
 tests/   Unit tests
 ├── Behat/Context  Some behat Context related to your module.
```

Comment:
```suggestion
```

Reply:
Left it on purpose to suggest that that's the place to put'em if needed though none exists currently. What do you think?

Reply:
Ok. let's leave both places.